sdv/constraints/tabular.py

```diff
@@ -401,3 +401,68 @@ def reverse_transform(self, table_data):
         table_data[self._column] = self._formula(table_data)
 
         return table_data
+
+
+class Rounding(Constraint):
+    """Round a column based on the specified number of digits.
+
+    Args:
+        columns (str or list[str]):
+            Name of the column(s) to round.
+        digits (int):
+            How much to round each column. All columns will be rounded to this
+            number of digits.
+        handling_strategy (str):
+            How this Constraint should be handled, which can be ``transform``
+            or ``reject_sampling``. Defaults to ``transform``.
+        tolerance (int):
+            When reject sampling, the sample data must be within this distance
+            of the desired rounded values.
+    """
+
+    def __init__(self, columns, digits, handling_strategy='transform', tolerance=None):
+        if digits > 15:
+            raise ValueError('The value of digits cannot exceed 15')
+
+        if tolerance is not None and tolerance >= 10**(-1 * digits):
+            raise ValueError('Tolerance must be less than the rounding level')
+
+        if isinstance(columns, str):
+            self._columns = [columns]
+        else:
+            self._columns = columns
+
+        self._digits = digits
+        self._tolerance = tolerance if tolerance else 10**(-1 * (digits + 1))
+        super().__init__(handling_strategy=handling_strategy, fit_columns_model=False)
+
+    def is_valid(self, table_data):
+        """Determine if the data satisfies the rounding constraint.
+
+        Args:
+            table_data (pandas.DataFrame):
+                Table data.
+
+        Returns:
+            pandas.Series:
+                Whether each row is valid.
+        """
+        rounded = table_data.round({column: self._digits for column in self._columns})
```

Comment:
We will need to restrict the validation to the constraint`[self._columns]`, since the other columns should not be compared.
Then, you can skip the dict and just use digits:

```python3
columns = table_data[self._columns]
rounded = columns.round(self._digits)
valid = (columns - rounded).abs() <= self._tolerance
```